.travis.yml

```diff
@@ -18,3 +16,34 @@ gemfile:
   - gemfiles/4.0.gemfile
   - gemfiles/4.1.gemfile
   - gemfiles/4.2.gemfile
+
+matrix:
+  include:
+    - rvm: 1.9.3
+      gemfile: gemfiles/1.9.3/3.0.gemfile
```

Comment:
You're removing 1.9.3 above, but it's removed from the travis setup above. Guess we can just go the route that the ruby-sdk goes: https://github.com/transloadit/ruby-sdk/commit/9da4fc50ffe5778b96b8fd43ca62f7830ca60efc